force-app/main/default/triggers/myAccountTrigger.trigger

```diff
@@ -0,0 +1,13 @@
+trigger myAccountTrigger on Account (before insert, after insert) {
```

Comment:
Triggers are still treated like classes and should be capitalized. You would just call this `AccountTrigger` or `OnAccount` or something like that.